ZipMod.py

```diff
@@ -37,32 +62,37 @@ def find_files(dir_path: str=None, patterns: [str]=None) -> [str]:
 
 # function to be passed to copytree, see https://docs.python.org/2/library/shutil.html#shutil.copytree
 def ignoreFunction(directory, files):
-  ignorePatterns = readGitIgnorePatterns()
+  ignorePatterns = readAllGitIgnorePatterns(modDir)
```

Comment:
I am not happy about using a global variable here. Would be better to curry the ignoreFunction, but I'm too tired to code that right now.